reliability/README.md

```diff
@@ -1,103 +1,41 @@
 # Drivy Reliability Challenge
 
-Looking for a job? Check out our [https://en.drivy.com/jobs](https://en.drivy.com/jobs).
+Looking for a job? Check out our [open positions](https://en.drivy.com/jobs).
+You can also take a look at our [engineering blog](https://drivy.engineering/) to learn more about the way we work.
 
 ## Guidelines
-- Clone this repo (do not fork it)
-- Commit your code
-- Once you are done, ping someone from Drivy (ideally via our jobs page)
 
-Please do the simplest thing that could work for the level you're currently solving.
+- Clone this repo (do **not** fork it)
+- Solve the levels in ascending order
+- Only do one commit per level and include the `.git` when submiting your test
 
-For higher levels we are interested in seeing code that is:
+## Pointers
 
-- Clean
-- Extensible
-- Reliable
+You can have a look at the higher levels, but please do the simplest thing that could work for the level you're currently solving.
 
-## Challenge
+The levels become more complex over time, so you will probably have to re-use some code and adapt it to the new requirements.
+A good way to solve this is by using OOP, adding new layers of abstraction when they become necessary and possibly write tests so you don't break what you have already done.
 
-The challenge needs to be resolved in Ruby.
+Don't hesitate to write [shameless code](http://red-badger.com/blog/2014/08/20/i-spent-3-days-with-sandi-metz-heres-what-i-learned/) at first, and then refactor it in the next levels.
 
-Each level is a directory containing ruby executables and libraries that you'll have to use.
+For higher levels we are interested in seeing code that is clean, extensible and robust, so don't overlook edge cases, use exceptions where needed, ...
 
-You can't modify them.
+Please also note that:
 
-Your solution to each level needs to live in the level directory.
+- Running `$ ruby main.rb` from the level folder should generate the desired output, but of course feel free to add more files if needed.
```

Comment:
Is this correct?

Reply:
The implementation for each level should live in a new file, `main.rb` seems standard enough.

Maybe we should provide empty `main.rb` files in each folder ?